release-tag.sh

```diff
@@ -0,0 +1,7 @@
+version=`python setup.py --version`
+eval "$(ssh-agent -s)" #start the ssh agent
+chmod 400 ./openfisca_bot
```

Comment:
Yes, it is a very strong security risk if you build PRs from forks. As an attacker, I just have to open a PR with a modified release script that prints the contents of the file.

You should rely on the security mechanisms from your CI engine. For example, CircleCI [disables many variables](https://circleci.com/docs/fork-pr-builds/) when building PRs from forks. If you add the SSH key through their web UI, everything will be handled for you.

I haven't looked up the equivalent with Travis, but I guess they have something too.